docs/src/Toast.doc.js

```diff
@@ -97,6 +102,56 @@ function ToastExample() {
   />
 );
 
+card(
+  <Example
+    id="redBackgroundColorExample"
+    name="Example: Red background color "
+    defaultCode={`
+function ToastExample() {
+  const [showToast, setShowToast] = React.useState(false);
+  return (
+    <Box>
+      <Button
+        inline
+        text={ showToast ? 'Close toast' : 'Show toast' }
+        onClick={() => setShowToast(!showToast)}
+      />
+      <Layer>
+        <Box
+          fit
+          dangerouslySetInlineStyle={{
+            __style: {
+              bottom: 50,
+              left: '50%',
+              transform: 'translateX(-50%)',
+            },
+          }}
+          paddingX={1}
+          position='fixed'
+        >
+          {showToast && (
+            <Toast
+              color="red"
+              text={
+                <>
+                  Saved to{' '}
```

Comment:
@Neet-Nestor Could we update this to be an error message instead? E.g. `Oops, we couldn't find that!`. It makes the intent more clear